src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -18,19 +18,21 @@
     "cudaVersion": {
       "type": "string",
       "enum": [
+        "11.8",
```

Comment:
Could we add a scenario (aka test) to confirm that the new values work as expected?

Reply:
Yes

Reply:
Added! How about a test in `test.sh`? This feature doesn't support `latest` for `cudaVersion`.

Reply:
Thanks for your feedback @HiddenLeafVill